examples/remote-import-class.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace remote_import;
+
+use Castor\Attribute\AsTask;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Criteria;
+use Doctrine\Common\Collections\Selectable;
+
+if (interface_exists(Selectable::class)) {
```

Comment:
it should not be need in most cases, but here we need it to make castor without remote task work